redeem/configuration/__init__.py

```diff
@@ -0,0 +1,23 @@
+try:  # TODO: remove when migrate to python 3
+    from configparser import ConfigParser
+except ImportError:
+    from ConfigParser import RawConfigParser as ConfigParser
+
+from redeem.configuration.factories.ConfigFactoryV19 import ConfigFactoryV19
+from redeem.configuration.factories.ConfigFactoryV20 import ConfigFactoryV20
+
+
+def get_config_factory(config_files):
+
+    config_parser = ConfigParser()
+    config_parser.read(config_files)
+
+    version = None
+    if config_parser.has_option('System', 'version'):
+        version = config_parser.get('System', 'version')
+
+    factory = ConfigFactoryV19()
```

Comment:
This is fine as-is, but in the long run, I think the factories need to be incremental and chainable. By that I meant that when we make more changes for 2.1, we don't want to have to go back and write those changes into the ConfigFactoryV19. If a user upgrades two versions, we really just want to use the ConfigFactoryV19 for the 1.9 -> 2.0 conversion and then write all the needed changes for 2.1 into ConfigFactoryV20.

This obviously isn't an immediate concern, but I think getting that sort of incremental model isn't possible the way the factories are currently written?